src/worker.jl

```diff
@@ -260,3 +267,15 @@ function _split_variables(scen_tree::ScenarioTree,
 
     return (branch_vars, leaf_vars)
 end
+
+function _execute_subproblem_callbacks(record::WorkerRecord,
+                                       niter::Int
+                                       )::Nothing 
+    for (scen, sub) in record.subproblems
```

Comment:
Shouldn't run over all subproblems. See above comment in `process_message(msg::Solve...`